### PR TITLE
MPDX-9433 frontend - MHI Eligibility UI improvements

### PR DESCRIPTION
**Pull request by zweatshirt**
## Description

- MHI Users need to be able to see their MHI amount (which is on MHA)
- MHI Eligibility needs to be explicitly shown on the MHA page

Follow up: Pass MHI or MHA status to SAA, so SAA knows to look up the correct payroll element.

## Testing

- Test with both regular MHA users and MHI users.
- Go to both the Salary Calculator, and the MHA request/calculator
- Check that MHI users can fill out an MHA amount (they are considered MHI)
- Check that MHI eligibility is displayed in MHA request/calculator dashboard, and only when the staff user is from Italy.

## Checklist:

- [x] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [x] I have applied the appropriate labels (_Add the label "Preview" to automatically create a preview environment_)
- [x] I have run the Claude Code `/pr-review` command locally and fixed any relevant suggestions
- [x] I have requested a review from another person on the project
- [x] I have tested my changes in preview or in staging
- [x] I have cleaned up my commit history

### CODE REVIEW

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/useMhaRequestData.test.tsx [hunk 1]
```diff
@@ -223,4 +223,51 @@ describe('useMhaRequestData', () => {
...
+
+    await expect(
+      result.current.schema.validate({
+        mhaAmount: 25000,
+        spouseMhaAmount: 0,
+      }),
+    ).rejects.toThrow(
+      'New Requested MHI cannot exceed Board Approved MHI Amount of $20,000',
+    );
+  });
```
Comment (zweatshirt) [inline src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/useMhaRequestData.test.tsx:272]: [Important] **Missing mixed-couple (Italian spouse) test coverage.** No test renders a US-user + Italian-spouse couple exercising the combined-max validation path, where `sectionKind` falls back to the spouse's kind (`useMhaRequestData.ts:60-61`). This is the most brittle part of the schema logic and the one place a regression would silently produce the wrong error copy. Suggested skeleton:

```tsx
it('uses the correct kind in combined-max validation for a mixed US/IT couple', async () => {
  const { result } = renderHook(() => useMhaRequestData(), {
    wrapper: createWrapper({
      hcmUser: { staffInfo: { country: 'US' }, mhaEit: { mhaEligibility: true } },
      hcmSpouse: {
        staffInfo: { country: 'IT' },
        mhaEit: { mhaEligibility: false },
        mhiEit: { mhiEligibility: true },
      },
    }),
  });
  await waitFor(() => expect(result.current.approvedAmount).toBe('$20,000'));
  await expect(
    result.current.schema.validate({ mhaAmount: 12000, spouseMhaAmount: 12000 }),
  ).rejects.toThrow(/Combined MHA amounts cannot exceed/);
});
```

⚠️ Important findings (7.0-7.9) prevent a CLEAN verdict but do not block merge. They cannot be dismissed via `/dismiss` — address in this PR or open a follow-up ticket.

File: src/components/HrTools/Shared/EligibilityStatusTable/EligibilityStatusTable.tsx [hunk 2]
```diff
@@ -54,10 +90,11 @@ export const EligibilityStatusTable: React.FC<EligibilityStatusTableProps> = ({
   const anyIneligible =
     !userEligible || (hasSpouse && spouseEligible === false);
   const anyEligible = userEligible || (hasSpouse && spouseEligible === true);
+  const showMhiRows = userCountry === 'IT' || spouseCountry === 'IT';
```
Comment (zweatshirt) [inline src/components/HrTools/Shared/EligibilityStatusTable/EligibilityStatusTable.tsx]: [Medium] **Inline `'IT'` checks duplicate the new `getHousingKind` helper.** This PR introduces `getHousingKind(country)` in `src/components/Reports/Shared/MinistersHousingEligibility/ministersHousingEligibility.ts` specifically to centralize the Italy→MHI rule, and `useMhaRequestData.ts` / `MinisterHousingAllowance.tsx` both adopt it. Same directory, different file — five raw `country === 'IT'` comparisons remain (`getMhiReason`, `showMhiRows`, two MHI cells, one at line 158). When the rule changes (new country, lowercase variant), half the call sites update and half drift. Per the user's "Consistent scoping of refactors" preference, this is the classic partial-refactor pattern to close out.

Suggested fix:

```tsx
import { getHousingKind } from '../MinistersHousingEligibility/ministersHousingEligibility';

const showMhiRows =
  getHousingKind(userCountry ?? null) === 'MHI' ||
  getHousingKind(spouseCountry ?? null) === 'MHI';
```

Apply analogous replacements at lines 68, 140, 148, and inside `getMhiReason`.

**Review by zweatshirt — COMMENTED** [review]
# Multi-Agent Code Review — PR #1739

**Verdict:** `APPROVED_WITH_SUGGESTIONS` — 0 blockers, 1 Important, 8 Medium, 2 Suggestions

**Risk (formula):** CRITICAL (score capped at 10). **Practical read:** HIGH. The formula is inflated by volume × scope multiplier on what is a focused MHI-eligibility UI PR with no auth, API, or data-integrity blockers. Substance of findings is consistent with APPROVED_WITH_SUGGESTIONS.

**Agents:** Architecture · Testing · Standards · Data Integrity · UX · Financial Reporting (domain agent from `.CLAUDE/rules/code-review.md`) — plus dependency-impact analysis and a full cross-examination debate round.

## Summary

No critical or high-priority blockers. The review surfaces:

- **1 Important finding** (severity 7.0): missing mixed-couple (Italian spouse) test coverage for combined-max validation — the gap allowed a now-resolved labeling-concern to go unnoticed.
- **8 Medium findings** (severity 5.0-6.9): `'IT'` magic-string duplication in `EligibilityStatusTable` vs. the new `getHousingKind` helper; one mild label-ambiguity in the mixed-couple MHA row (post-debate consensus: display-layer confusion, not a data bug); ambiguous `N/A` cells and italic-only paper-form note for screen readers; `'USD'` hardcoded where Italian MHI amounts will now render.
- **2 Suggestions** (severity < 5.0): comment wording, and a style-consistency note on kind-interpolation.

⚠️ **Important findings (7.0-7.9)** cannot be dismissed via `/dismiss` (threshold is 7.0). They do not block merge but also do not permit a CLEAN verdict — address them in this PR or open a follow-up ticket.

## Debate round highlights

- **Data Integrity DI1 (originally 7.5)** — DI agent CONCEDED after re-reading: `getEffectiveEligibility` is consistently applied; the MHA row showing "Eligible" for an Italian MHI-eligible spouse is a label phrasing issue, not a contradiction. Revised to 5.0. UX agent confirmed the behavior is intentional for the unified-request workflow.
- **Financial F1 (originally 7.0)** — REVISED to 6.0. The `'USD'` hardcode predates this PR; this PR doesn't regress it, only exposes the gap. Appropriate for a follow-up ticket.
- **Testing T2 (originally 6.5)** — RAISED to 7.0 after confirming no mixed-couple `EligibilityStatusTable` test exists in `MhaRequestSection.test.tsx`. This is the sole Important-tier finding.
- **Standards S2 (originally 5.0)** — DROPPED to 2.0. Generated types confirm `staffInfo` is non-nullable on `HcmQuery['hcm'][number]`; the removed `?.` was redundant, not a safety relaxation.

## Cross-Cutting Consistency Check

Two code paths render the same `EligibilityStatusTable`:

| Tree | Props | Data source |
|---|---|---|
| `MinisterHousingAllowance.tsx` (report) | `userEligible` = **raw** `mhaEligibility` | Correct per the documented bifurcation — Italians are blocked from the online MHA flow |
| `MhaRequestSection.tsx` (SalaryCalc) | `userEligible` = **effective** eligibility | Intentional — the section only renders when some program qualifies the user |

The prop-semantics divergence is intentional per UX review but surfaces the mild label-ambiguity flagged in DI1 (mixed couple with Italian spouse). Documenting this invariant on `EligibilityStatusTable`'s prop types would reduce future drift.

## Dependency Impact

- **High impact, no breakage:** `MinisterHousingAllowanceContext.tsx` (13 consumers) narrowed `HcmData | undefined` → `HcmData | null` — all existing consumers use optional chaining, so the narrowing is type-only.
- **Additive:** `Hcm.graphql` adds `mhiEit { mhiEligibility }` — 7 consumers of `useHcmQuery`, no schema-breaking change.
- **New files:** `ministersHousingEligibility.ts` (2 consumers), `EligibilityStatusTable` prop additions (2 consumers) — well-contained.

## Review Summary Table

| Agent | Critical | High | Important | Medium | Suggestions | Confidence |
|-------|---|---|---|---|---|---|
| Security | — | — | — | — | — | N/A (skipped — no auth/API/workflow surface) |
| Architecture | 0 | 0 | 0 | 1 | 1 | High |
| Testing | 0 | 0 | 1 | 1 | 1 | High |
| Standards | 0 | 0 | 0 | 0 | 2 | High |
| Data Integrity | 0 | 0 | 0 | 2 | 1 | Medium-High |
| UX | 0 | 0 | 0 | 4 | 0 | Medium-High |
| Financial Reporting | 0 | 0 | 0 | 1 | 0 | Medium-High |
| **Total** | **0** | **0** | **1** | **9** | **5** | |

## Verification Context

Working tree clean on head `d6af5ad22`. `yarn test` on the affected trees: 317/317 pass. `yarn lint:ts` clean.

---

File: src/components/HrTools/Shared/EligibilityStatusTable/EligibilityStatusTable.tsx [hunk 3]
```diff
@@ -92,8 +131,50 @@ export const EligibilityStatusTable: React.FC<EligibilityStatusTableProps> = ({
...
+          {showMhiRows && (
+            <>
+              <MhiSectionRow>
+                <TableCell>{t('MHI Eligibility')}</TableCell>
+                <TableCell>
+                  {userCountry === 'IT'
+                    ? userMhiEligibility
+                      ? t('Eligible')
+                      : t('Ineligible')
+                    : t('N/A')}
```
Comment (zweatshirt) [inline src/components/HrTools/Shared/EligibilityStatusTable/EligibilityStatusTable.tsx]: [Medium] **Ambiguous `N/A` for screen readers.** Plain `N/A` cells announce as "en-slash-ay" without context — users don't know whether it means "not applicable," "not available," or "unknown." For a mixed US/IT couple, the non-Italian partner's MHI cells render `N/A`. Consider a contextual `aria-label`, e.g.:

```tsx
<TableCell aria-label={t('MHI not applicable — {{name}} is not Italy staff', { name: userPreferredName })}>
  {t('N/A')}
</TableCell>
```

Or use `t('Not applicable')` for the visible text.

File: src/components/Reports/Shared/HousingAllowance/housingAllowance.test.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    expect(getEffectiveEligibility(usHcm)).toBe(true);
+    expect(
+      getEffectiveEligibility({
+        staffInfo: { country: 'US' },
+        mhaEit: { mhaEligibility: false },
+        mhiEit: { mhiEligibility: false },
+      }),
+    ).toBe(false);
+  });
+});
```
Comment (zweatshirt) [inline src/components/Reports/Shared/HousingAllowance/housingAllowance.test.ts:59]: [Medium] **Missing test: `getEffectiveEligibility(null)`.** `getMhiEligibility(null)` is tested (line 36) but the null-hcm path through `getEffectiveEligibility` is not. `MinisterHousingAllowanceContext` now defaults `userHcmData` to `null` (not `undefined`), so the null path is reachable in production. Low-effort addition:

```ts
it('returns false when hcm is null', () => {
  expect(getEffectiveEligibility(null)).toBe(false);
});
```

File: src/components/HrTools/Shared/EligibilityStatusTable/EligibilityStatusTable.tsx [hunk 5]
```diff
@@ -10,17 +10,38 @@ import {
...
+  '.MuiTableCell-head': {
+    fontWeight: 'bold',
+    color: theme.palette.primary.main,
+  },
+  '.MuiTableBody-root .MuiTableCell-root:first-of-type': {
+    fontWeight: 'bold',
+  },
+}));
+
+const MhiSectionRow = styled(TableRow)(({ theme }) => ({
```
Comment (zweatshirt) [inline src/components/HrTools/Shared/EligibilityStatusTable/EligibilityStatusTable.tsx:30]: [Medium] **MHI section divider is color/visual only — not exposed to assistive tech.** `MhiSectionRow` uses a 2px `borderTop` to separate MHA from MHI rows. Screen readers get no indication of the row grouping. Consider wrapping MHA rows and MHI rows in labeled `<tbody>` groups (with `aria-label`) or adding a sub-header row ("MHI" in a header cell) so the grouping is structural rather than decorative.

File: src/components/HrTools/Shared/EligibilityStatusTable/EligibilityStatusTable.tsx [hunk 6]
```diff
@@ -116,6 +197,22 @@ export const EligibilityStatusTable: React.FC<EligibilityStatusTableProps> = ({
           </Typography>
         </Box>
       )}
+      {showMhiRows && (
+        <Box
+          sx={{ mt: compact ? theme.spacing(1) : theme.spacing(2) }}
+          data-testid="mhi-paper-form-note"
```
Comment (zweatshirt) [inline src/components/HrTools/Shared/EligibilityStatusTable/EligibilityStatusTable.tsx:206]: [Medium] **Paper-form note lacks semantic weight.** The MHI paper-form guidance is the primary actionable message for Italian staff, but it's rendered as italic `Typography` with no `role`, icon, or landmark. Promote to either `role="note"` on the outer Box, or use `<Alert severity="info">` with an `InfoIcon` — this makes it scannable visually and announceable to screen readers as a distinct region rather than a footnote.

```tsx
<Alert severity="info" icon={<InfoOutlinedIcon />} data-testid="mhi-paper-form-note">
  <Trans t={t}>Note: Italy staff must complete a paper MHI form.</Trans>
</Alert>
```

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx [hunk 7]
```diff
@@ -93,53 +153,66 @@ export const MhaRequestSection: React.FC = () => {
...
-                href={`/accountLists/${accountListId}/hrTools/mhaCalculator`}
-              >
-                this link
-              </Link>
-              . Pending MHA Requests will not apply to this salary calculation
-              but a new Salary Calculation Form can be submitted after approval.
-            </Trans>
+            {sectionKind === 'MHI' ? (
+              <Trans t={t}>
+                Our records show that not all staff have an MHI for the
```
Comment (zweatshirt) [inline src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx:139]: [Medium] **"have an MHI" reads awkwardly.** MHI expands to "Minister's Housing Italy" — mentally substituting produces "have a Minister's Housing Italy," which parses as a missing noun. The parallel MHA copy (line 172) reads cleanly ("have a Minister's Housing Allowance"). Also: the MHI branch never expands the acronym anywhere, so Italian users have no in-product context for what MHI means. Consider:

- `have an MHI approval` / `have an approved MHI`
- or `have a Minister's Housing Italy (MHI) approval`

This is worth a product/UX sign-off before final merge.

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/useMhaRequestData.ts [hunk 8]
```diff
@@ -36,6 +43,10 @@ export const useMhaRequestData = () => {
     (userEligible && !userHasApprovedMha) ||
     (spouseEligible && !spouseHasApprovedMha);
 
+  // For Italian (MHI) staff the backend stores the approved MHI amount in
+  // `mhaRequest.currentApprovedOverallAmount` — there is no separate
+  // `mhiRequest` field. This one value is the approved amount for whichever
+  // kind (MHA or MHI) applies to the person.
   const approvedAmount = userHasApprovedMha
     ? (hcmUser?.mhaRequest.currentApprovedOverallAmount ?? 0)
     : (hcmSpouse?.mhaRequest.currentApprovedOverallAmount ?? 0);
```
Comment (zweatshirt) [inline src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/useMhaRequestData.ts:52]: [Medium] **Hardcoded `'USD'` now exposed to Italian MHI users (re: line 55 and four other call sites).** Pre-existing pattern — this PR doesn't regress it — but Italian MHI staff now reach this code path, and the existing test at `useMhaRequestData.test.tsx:260` explicitly expects `'$20,000'` for an Italian user. If the backend stores Italian MHI amounts in EUR, every user on the MHI path sees a `$` prefix on a EUR-valued amount.

Five call sites hit `'USD'` (lines 55, 154, 158, 162, 166). Recommend a follow-up ticket to:
1. Confirm with backend the currency contract for `mhaRequest.currentApprovedOverallAmount` per country
2. Thread the correct currency code through (either from API or derived from country)

Appropriate for a follow-up ticket — not a merge blocker.

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx [hunk 9]
```diff
@@ -73,16 +128,21 @@ export const MhaRequestSection: React.FC = () => {
...
-      <CardHeader title={t('MHA Request')} subheader={<EffectiveDateNote />} />
+      <CardHeader
+        title={sectionKind === 'MHI' ? t('MHI Request') : t('MHA Request')}
+        subheader={<EffectiveDateNote />}
+      />
       <CardContent>
         {anyIneligible && (
           <EligibilityStatusTable
             userPreferredName={userPreferredName}
             userEligible={userEligible}
```
Comment (zweatshirt) [inline src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx:120]: [Medium] **Mild label ambiguity in mixed-couple eligibility table.** `userEligible` and `spouseEligible` are the **effective** eligibility (MHI flag for Italian staff) from `useMhaRequestData`. When `EligibilityStatusTable` renders them under a row labeled "MHA Eligibility" (triggered when any country is IT via `showMhiRows`), the Italian spouse's cell reads "Eligible" despite their raw `mhaEligibility` being false. 

Post-debate consensus: this is intentional (the SalaryCalc section only renders when at least one party qualifies via some program), but the row label being "MHA Eligibility" rather than "Eligibility" creates mild confusion. 

Contrast with `MinisterHousingAllowance.tsx` (line 166) which passes **raw** `mhaEligibility` to the same component — so the two trees read the same table differently. Document this semantic split on `EligibilityStatusTable`'s prop types (e.g., a comment on `userEligible` / `spouseEligible` clarifying it means "effective eligibility for the housing program that applies"), or consider renaming the prop to `userHousingEligible` to reduce drift risk.

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/useMhaRequestData.ts [hunk 10]
```diff
@@ -45,39 +56,53 @@ export const useMhaRequestData = () => {
     [approvedAmount, locale],
   );
 
+  // Validation messages follow the same per-kind labeling as the UI.
+  const sectionKind =
+    showSpouseFields && !showUserFields ? spouseKind : userKind;
+
   const schema = useMemo(() => {
     const combinedMaxMessage = t(
```
Comment (zweatshirt) [inline src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/useMhaRequestData.ts:64]: [Suggestion] **Inconsistent kind-dispatch style.** This file uses i18next interpolation `t('Combined {{kind}} amounts cannot exceed …', { kind: sectionKind })` (lines 64, 68, 73, 78, 83) while the adjacent `MhaRequestSection.tsx` uses explicit string branches `sectionKind === 'MHI' ? t('Combined MHI Requested') : t('Combined MHA Requested')`. Both are valid i18next patterns and both are extractable. Not a violation — but consistent style across the two files would reduce cognitive overhead and give translators a single model to work from.

File: src/components/Reports/Shared/HousingAllowance/housingAllowance.ts [hunk 11]
```diff
@@ -0,0 +1,22 @@
+export type HousingKind = 'MHA' | 'MHI';
+
+// Structural shape: both the shared HcmData query and SalaryCalculator's own
```
Comment (zweatshirt) [inline src/components/Reports/Shared/HousingAllowance/housingAllowance.ts:3]: [Suggestion] **Misleading comment.** The comment says "both the shared HcmData query and SalaryCalculator's own Hcm query include these fields" — but there's only one `Hcm.graphql`, which SalaryCalculator also consumes via `useHcmQuery` imported from `../../Shared/HcmData/Hcm.generated`. Consider:

```ts
// Structural shape: the shared Hcm query (src/components/Reports/Shared/HcmData/Hcm.graphql)
// includes these fields; all consumers derive their types from that query.
```

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx (hunk 7 above)
Comment (zweatshirt) [reply src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx:139]: If we knew what MHI means we could change this. Claude just assumes the 'I' is for 'Italy.' Maybe it is, not sure.

**Comment by zweatshirt** [comment]
@canac Agent PR review came back critical. The biggest thing is just ensuring the UI works as expected. I will have a follow up backend PR soon, since SAA needs to know whether the user submitted a Salary Request using MHI or MHA.

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx [hunk 12]
```diff
@@ -61,10 +61,65 @@ export const MhaRequestSection: React.FC = () => {
...
 
+  // Section-level copy follows the primary user's kind, except when only the
+  // spouse's fields show — then it follows the spouse's kind so an MHI-only
+  // spouse on a mixed-country couple gets MHI wording.
+  const sectionKind =
+    showSpouseFields && !showUserFields ? spouseKind : userKind;
+
+  const boardApprovedText = () => {
+    if (showUserFields && showSpouseFields) {
+      return sectionKind === 'MHI'
```
Comment (canac) [inline src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx]: Should we interpolate `sectionKind` instead of branching? I think it's OK if MHA/MHI isn't translated because it's an abbreviation.

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx [hunk 13]
```diff
@@ -93,53 +153,66 @@ export const MhaRequestSection: React.FC = () => {
...
-              </Link>
-              . Pending MHA Requests will not apply to this salary calculation
-              but a new Salary Calculation Form can be submitted after approval.
-            </Trans>
+            {sectionKind === 'MHI' ? (
+              <Trans t={t}>
+                Our records show that not all staff have an MHI for the
+                effective date of this salary calculation. If an MHI Request
+                form has not yet been submitted, it may be completed using{' '}
+                <Link
```
Comment (canac) [inline src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx]: But they can't submit MHI requests at that link, right?

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx [hunk 14]
```diff
@@ -93,53 +153,66 @@ export const MhaRequestSection: React.FC = () => {
...
               </Trans>
             </Typography>
             <Typography variant="body1">
-              <Trans t={t}>
-                Please enter the amount of your salary you would like to request
-                as MHA below. If you have a pending MHA Request for a new
-                amount, it will not apply to this salary calculation but you can
-                submit a new Salary Calculation Form after it is approved.
-              </Trans>
+              {sectionKind === 'MHI' ? (
```
Comment (canac) [inline src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx]: We could consider interpolating `sectionKind` here too.

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/useMhaRequestData.ts [hunk 15]
```diff
@@ -36,6 +43,10 @@ export const useMhaRequestData = () => {
     (userEligible && !userHasApprovedMha) ||
     (spouseEligible && !spouseHasApprovedMha);
 
+  // For Italian (MHI) staff the backend stores the approved MHI amount in
```
Comment (canac) [inline src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/useMhaRequestData.ts:46]: praise: This is a helpful comment!

File: src/components/HrTools/Shared/EligibilityStatusTable/EligibilityStatusTable.tsx [hunk 16]
```diff
@@ -78,7 +120,7 @@ export const EligibilityStatusTable: React.FC<EligibilityStatusTableProps> = ({
             )}
           </TableRow>
           <TableRow>
-            <TableCell>{t('Reason')}</TableCell>
+            <TableCell>{showMhiRows ? t('MHA Reason') : t('Reason')}</TableCell>
```
Comment (canac) [inline src/components/HrTools/Shared/EligibilityStatusTable/EligibilityStatusTable.tsx:123]: praise: I like that this keeps it simpler for most non-Italy staff!

**Review by canac — APPROVED** [review]
I haven't tested with MHI users, but I can if that would be helpful. Which test users do you use?

**Comment by zweatshirt** [comment]
> I haven't tested with MHI users, but I can if that would be helpful. Which test users do you use?

The users with @agapeitalia.org email domain. Might be a good idea to test once I am done fully integrating. Almost there as I am finishing up my SAA PR now

File: src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx (hunk 13 above)
Comment (zweatshirt) [reply src/components/HrTools/SalaryCalculator/YourInformation/MhaRequestSection/MhaRequestSection.tsx]: True, I'll try to make that clear for MHI users
